lasagne/nonlinearities.py

```diff
@@ -286,6 +286,71 @@ def softplus(x):
     return theano.tensor.nnet.softplus(x)
 
 
+# Clipped Activation
+class ClippedActivation(object):
+    """Clipped Activation activation function
```

Comment:
What about `Clip an activation function`? The duplicate `Activation activation` reads funny.